### PR TITLE
fixing setup,reference,aio

### PR DESCRIPTION
**Pull request by hoytpr**
Made simple changes to YAML headers to allow setup.md,
reference.md, and aio.md, to properly render the CSS
and to correct the navigation from these pages.
Might be applicable to other lessons where more extreme
changes have been made or proposed.

Changed pages can be seen at:
https://hoytpr.github.io/organization-genomics-test/setup.html
https://hoytpr.github.io/organization-genomics-test/aio.html
https://hoytpr.github.io/organization-genomics-test/reference.html

### CODE REVIEW

**Comment by hoytpr** [comment]
I spent a lot of time on this and believe it is correct now. The pages render CSS and the links now work from within those pages. The reference page doesn't have any references... but at least it's working. This fixes issues #76 and #75 and #73. 
It may also fix issues with other lessons
(e.g. maybe datacarpentry/shell-genomics#169).

File: aio.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
-layout: page
```
Comment (fmichonneau) [inline aio.md:2]: please keep the `page` layout (otherwise a blank overview appears on top of the page)

**Review by fmichonneau — CHANGES_REQUESTED** [review]
could you please remove the `permalink` line altogether?

File: aio.md (hunk 1 above)
Comment (hoytpr) [reply aio.md:2]: Done. Sorry it didn't seem to work last night,

**Comment by hoytpr** [comment]
changes made. hope it's correct now. Seems correct on the test site links shown above. 
Did you want ALL the permalinks removed?  
Okay. All permalinks removed.
All pages using 'page' layout now.

File: aio.md (hunk 1 above)
Comment (hoytpr) [reply aio.md:2]: All pages now using page layout and have permalink removed.

File: reference.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-layout: reference
-permalink: /reference/
+layout: page
```
Comment (fmichonneau) [inline reference.md]: you want to keep layout to `reference` here otherwise the page won't look quite right.

**Comment by fmichonneau** [comment]
you need to fix the value of one layout variable but otherwise it looks good to me!

File: reference.md (hunk 2 above)
Comment (hoytpr) [reply reference.md]: done. Sorry for the confusion.

**Comment by fmichonneau** [comment]
Thanks!

**Pull request merged**